app/src/main/java/com/simplecity/amp_library/ui/detail/album/AlbumDetailPresenter.kt

```diff
@@ -44,27 +44,21 @@ class AlbumDetailPresenter constructor(private val album: Album) : Presenter<Alb
     }
 
     fun fabClicked() {
-        MusicUtils.shuffleAll(songs) { message ->
-            view?.showToast(message)
-        }
+        musicUtils.shuffleAll(songs, { view?.showToast(it) })
```

Comment:
Although this is more concise, the convention in Shuttle is to prefer not to use implicit naming of lambda parameters. (Prefer `message` to `it`).